### PR TITLE
Zgłoszenie rozwiązania

### PR DESCRIPTION
**Pull request by ShouldIStayOrShouldIGo**

### CODE REVIEW

File: src/main/java/edu/kis/vh/nursery/FIFORhymer.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends DefaultCountingOutRhymer {
+public final class FIFORhymer extends DefaultCountingOutRhymer {
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/FIFORhymer.java]: There was no task concerning class modifiers

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 2]
```diff
@@ -1,21 +1,21 @@
 package edu.kis.vh.nursery.list;
 
-public class IntLinkedList {
+public final class IntLinkedList {
 
-    Node last;
-    int i;
+    private Node last;
+    private int i;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java:7]: Unused private field could be removed.

File: src/main/java/edu/kis/vh/nursery/list/Node.java [hunk 3]
```diff
@@ -1,12 +1,35 @@
 package edu.kis.vh.nursery.list;
 
-public class Node {
+final class Node {
 
-    public int value;
-    public Node prev, next;
+    private int value;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/Node.java]: Immutable - should be final (setter unnecessary)

**Review by kowallus — CHANGES_REQUESTED** [review]
A crucial part of lab. is missing. See 1.4 and "Temat: stałe i annotacje"
Review hints: tiny.cc/2rkvgz

**Comment by ShouldIStayOrShouldIGo** [comment]
Files have been changed. Corrections in commented files

**Comment by kowallus** [comment]
-0.5 pts

**Pull request merged**